t/01-pdk/12-node/02-get_memory_stats.t

```diff
@@ -436,3 +436,47 @@ GET /t
 workers_lua_vms\Z
 --- no_error_log
 [error]
+
+
```

Comment:
missing blank line (you can run the `reindex` script for this) (https://github.com/openresty/openresty-devel-utils)

Reply:
Thanks @thibaultcha - did not know about that tool!